dictdiffer/utils.py

```diff
@@ -16,6 +16,15 @@
 EPSILON = sys.float_info.epsilon
 
 
+class DottedIgnoreKey(str):
+    """Custom type to specify dotted ignore key.
+
+    This custom type help to desactivate dotted interpretation.
+    """
+
+    pass
```

Comment:
It's not necessary when the `class` has a docstring.
 
```suggestion
    pass
```

Reply:
Oki good to know, thx :)